sdk/rust/oak/src/grpc/server.rs

```diff
@@ -52,18 +52,18 @@ pub fn init_with_sender(
     invocation_sender: Sender<Invocation>,
 ) -> Result<(), OakStatus> {
     let config = crate::node_config::grpc_server(address);
+    // TODO(#1631): When we have a separate top for each sub-lattice, this should be changed to
+    // the top of the identity sub-lattice.
+    let top_label = oak_abi::label::confidentiality_label(oak_abi::label::top());
     // Create a channel and pass the read half to a new gRPC server pseudo-Node.
-    let init_sender = match crate::io::node_create::<GrpcInvocationSender>(
-        "grpc_server",
-        &Label::public_untrusted(),
-        &config,
-    ) {
-        Ok(s) => s,
-        Err(e) => {
-            let _ = invocation_sender.close();
-            return Err(e);
-        }
-    };
+    let init_sender =
+        match crate::io::node_create::<GrpcInvocationSender>("grpc_server", &top_label, &config) {
```

Comment:
Just to clarify... previously the gRPC and the HTTP server nodes where automatically doing declassifying reads using their privileges. Is it right? 
With this change, do the `GrpcServerNode` and `HttpServerNode` still need to have "top" as their privilege?

Reply:
Yes, they previously did declassifying reads which they no longer need to do because they are now top secret. But now they do declassifying writes to send the invocations from a top secret node to less secret nodes, so they still need the privilege.

Reply:
That is right. Thanks.